packages/vite-extensions/vite-website-cache-paths.ts

```diff
@@ -0,0 +1,77 @@
+import { readdirSync, statSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+
+const patternsToCache = [
```

Comment:
This seems fragile – it will break the moment a new asset file is added. Is there any non-PHP file that we do **not** want to cache here?

Reply:
+1 I don't think that this is great, but there is a lot of files that we don't want to download. 

>  Is there any file in assets/ that we do not want to cache?

Yes, if you go to `dist/packages/playground/wasm-wordpress-net` you can see that there are many more files.


I'm not sure what's the best approach here. If I create a blacklist it will have the same fragile issue but now instead of offline support breaking it will cache files the user doesn't need.

I think that the best solution for now would be to keep the current implementation but add a test to ensure we catch any errors.

What do you think?

Reply:
E2E tests will be tricky and could require a lot of work.

Our current test runner uses the local development environment which doesn't support offline mode.

We could modify E2E tests to use a preview server instead of the local development environment.

Reply:
I will work on a blacklist to move this forward and we can revisit E2E tests later.

Reply:
This commit switches to a exclude list [a746657](https://github.com/WordPress/wordpress-playground/pull/1573/commits/a746657ff67d02b1fd7018e2ea7c88ad134a46f9)